eva_submission/nextflow/variant_load.nf

```diff
@@ -45,34 +45,33 @@ if (!params.valid_vcfs || !params.project_accession || !params.load_job_props ||
 
 /*
 If the aggregation type is NONE (genotyped VCF) the csv file with the mapping between vcf file, assembly accession,
-fasta, assembly report, analysis_accession, db_name will be grouped by analysis accession and the vcf files per
-analysis will be counted to determine if a merge is needed. When merge is not needed a symbolic link will be created
+fasta, assembly report, analysis_accession, db_name, aggregation will be grouped by analysis accession and the vcf files
+per analysis will be counted to determine if a merge is needed. When merge is not needed a symbolic link will be created
 to the input vcf file
-If the aggregation type is different from none (SIMPLE = aggregated VCF, with allele frequencies) the files are not
+If the aggregation type is different from none (BASIC = aggregated VCF, with allele frequencies) the files are not
 merged and passed directly to create the properties
 **/
-(vcfs_to_merge, unmerged_vcfs) = (
-    params.aggregation_type.toLowerCase().equals("none")
-    ? [Channel.fromPath(params.valid_vcfs)
+vcfs_to_merge = Channel.fromPath(params.valid_vcfs)
```

Comment:
Maybe can use [`branch`](https://www.nextflow.io/docs/latest/operator.html#branch) here?  Not sure it's much more readable than this, up to you.

Reply:
That would make sense if the two object were using the same data structure. But here they are different so it would make the rest of the operation quite complicated. 
I didn't know about branch though, so thanks ...